framework/source/class/qx/event/handler/GestureCore.js

```diff
@@ -91,6 +121,12 @@ qx.Bootstrap.define("qx.event.handler.GestureCore", {
     __initialDistance : null,
     __momentum : null,
 
+    __rollEvents : null,
+    __rollEventsCountStart : null,
+    __rollEventsCount : null,
+    __touchPadDetectionPerformed : null,
+    __lastRollEventTime: null,
```

Comment:
rollEventsCountStart, rollEventsCount, touchPadDetectionPerformed, and last RollEventTime are all intended to hold non-reference types. They should not be initialized to null. Rather, they should be initialized to 0 for the numeric ones, and false for the boolean one... and then there's no need (although it doesn't hurt) to initialize them in the constructor.